Update how-to-monitor-model-performance.md (#115824)
Azure/azure-content: articles/machine-learning/how-to-monitor-model-performance.md

### Component output signature

The component output port should have the following signature.

  | signature name | type | description | 
  |---|---|---|

>   | signal_metrics | mltable | The ml table which contains the computed metrics. The schema is defined in the signal_metrics schema section below. |

Reviewer: ```suggestion
  | signal_metrics | mltable | The ml table that contains the computed metrics. The schema is defined in the signal_metrics schema section in the next section. |
```
